modules/inline-editable/main/index.scss

```diff
@@ -4,4 +4,8 @@
   .hx-input-group {
     margin: -0.45em 0;
   }
+}
+
+.hx-inline-editable-no-value {
+  font-style: italic;
```

Comment:
Making it italic when no value has been entered is probably enough of a visual cue (along with the `Enter Value` text) that the user should do something